README.md

```diff
@@ -1 +1,61 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <img src="logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> This is Day 2 project for learning Git workflows. I aim to follow the instructions as mentioned in the task board and set up this repo.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> This project catered to understand how to create a repo. Create branches with different base branches. Understand the concept of version control. Follow GitHub flow. Open pull requests in GitHub. Write descriptive, easy to understand commit messages. Write short, detailed, easy to understand descriptions of pull requests. Write a descriptive and easy to understand README in business English for every project submitted. Demonstrate an ability to submit a project for code review.
+
+- **[Follow GitHub flow]**
+- **[Open pull requests in GitHub]**
+- **[Demonstrate an ability to submit a project for code review]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Live link is coming soon..
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+> 
+
+👤 **Sandeep**
+
+- GitHub: [@mailsg](https://github.com/mailsg)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] as requested by the previous reviewer kindly consider adding a working link to your MIT license this will improve the user experience for the people visiting your project